src/platform/site-wide/announcements/components/MissionAct.jsx

```diff
@@ -5,7 +5,7 @@ export default function MissionAct({ dismiss }) {
     <div className="personalization-announcement">
       <span className="usa-label va-label-primary">New</span>{' '}
       <a onClick={dismiss} href="https://missionact.va.gov/">
-        Learn how you can get easier access to health care with the MISSION Act.
+        Learn how you can get easier access to health care with the MISSION Act
```

Comment:
Follow up to https://github.com/department-of-veterans-affairs/vets-website/pull/10175